### PR TITLE
CDAP-7024 Increase margin of delegation token update from 5 minutes to 1 hour

### PR DESCRIPTION
**Pull request by anwar6953**
CDAP-7024 Increase margin of delegation token update from 5 minutes to 1 hour.
Currently, if there are a lot of programs running, and we want to update all of their tokens, it may take more than 5 minutes (or if there's some slowness). Doing it at the 5 minute marker before expiry is risky.
This change helps avoid that risk.

https://issues.cask.co/browse/CDAP-7024
http://builds.cask.co/browse/CDAP-RUT357-1

### CODE REVIEW

File: cdap-app-fabric/src/main/java/co/cask/cdap/security/TokenSecureStoreUpdater.java [hunk 1]
```diff
@@ -226,6 +225,9 @@ private long calculateUpdateInterval() {
    * @return The update interval in milliseconds.
    */
   public long getUpdateInterval() {
+    if (updateInterval == null) {
```
Comment (albertshau) [inline cdap-app-fabric/src/main/java/co/cask/cdap/security/TokenSecureStoreUpdater.java:228]: why the change to set lazily versus calculating in the constructor?

Comment (anwar6953) [reply cdap-app-fabric/src/main/java/co/cask/cdap/security/TokenSecureStoreUpdater.java:228]: sometimes, we don't schedule it (so we dont call getUpdateInterval), but simply call its update method (such as during launching of programs).
So, when launching 100 workflows, it will no longer log 100x the log message at the bottom of computeUpdateInterval(), which can be misleading.

Comment (albertshau) [reply cdap-app-fabric/src/main/java/co/cask/cdap/security/TokenSecureStoreUpdater.java:228]: can you add a comment explaining

**Review by albertshau — APPROVED** [review]
lgtm

File: cdap-app-fabric/src/main/java/co/cask/cdap/security/TokenSecureStoreUpdater.java (hunk 1 above)
Comment (anwar6953) [reply cdap-app-fabric/src/main/java/co/cask/cdap/security/TokenSecureStoreUpdater.java:228]: added

**Comment by anwar6953** [comment]
Build was green; simply added a comment, so not rerunning build for that.
